phantom-connectors/src/main/scala/com/websudos/phantom/connectors/Keyspace.scala

```diff
@@ -87,11 +87,9 @@ class KeySpaceDef(
   clusterBuilder: ClusterBuilder,
   autoinit: Boolean,
   keyspaceFn: Option[(Session, KeySpace) => String] = None
-) {
+) { outer ⇒
```

Comment:
@polymorphic Could you replace this with a standard ASCII based `=>`? Our static analysis tools are complaining, for consistency purposes we don't use fancier versions of characters just yet.